src/stirling/bpf_tools/bcc_symbolizer_bpf_test.cc

```diff
@@ -68,7 +68,8 @@ TEST(BCCSymbolizer, ModuleName) {
   for (const auto& entry : smaps) {
     if (entry.pathname == "[vdso]") {
       const std::string_view symbol = symbolizer.SymbolOrAddrIfUnknown(pid, entry.vmem_start);
-      EXPECT_EQ(symbol, "[m] [vdso] + 0x00000000");
+      const std::string expected = absl::StrCat("[m] [vdso] + 0x", absl::Hex(entry.vmem_start));
+      EXPECT_EQ(symbol, expected);
```

Comment:
https://github.com/iovisor/bcc/pull/4319 provides better symbolization and is the source of this behavior change. I verified this by reverting the change and seeing if the previous version of the test succeeded.